---
layout: minimal_default
title: Yuzhe's Homepage
noindex: True
---
<h1>Research</h1>
<p>This is a list of my research projects.</p>

<div class="media">
  <a class="pull-left" href="#">
    <img class="media-object hidden-phone" src="static/yyz.jpg" width="96px">
  </a>
  <div class="media-body">
    <p class="media-heading">
      <strong><i>De Novo</i> DNA Sequence Assembly from Barcoded Reads</strong> <a href="assets/pdf/10x_poster.pdf">[Poster]</a> <br>
      Fei Xia, <a href="http://www.ilanshomorony.com">Ilan Shomorony</a>, <a href="https://web.stanford.edu/~dntse/"> David Tse</a>. 
      
    </p>
    <p class="abstract-text">
	<p>We introduce necessary and sufficient conditions for a given repeat in the genome to be resolved by a barcoded fragment, which translates into minimum coverage requirements for perfect assembly as a function of the sequence repeat statistics. Based on these theoretical insights, we develop a new algorithm to perform de novo assembly from barcoded short reads. The algorithm takes a two-step approach. First, it uses the raw reads with no barcode information to construct a reliable de Bruijn graph. Then, the barcodes are mapped onto this graph and used to resolve repeats, leading to an increase in the resulting contig lengths. We quantify the impact of barcodes by studying the de Bruijn graph degree profile, N50, and maximum contig lengths before and after the repeat resolution step. Experiments on synthetic datasets and human genome datasets show the effectiveness of our algorithm.</p>
    </p> 
  </div>
</div>

